sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -117,9 +140,14 @@ function buildAttributes(attrs: { [key: string]: { toString(): string } }): Attr
 }
 
 function buildNode(obj: any, elementName: string): Node[] {
-  if (typeof obj === "string" || typeof obj === "number" || typeof obj === "boolean") {
+  if (
+    obj == undefined ||
+    typeof obj === "string" ||
+    typeof obj === "number" ||
+    typeof obj === "boolean"
```

Comment:
We are missing tests for the case that has been added here for undefined.
I reverted this change and expected some test to fail, none failed

Reply:
Given no tests exist for existing code on the string/number/boolean checks, it seems odd to have a test just for undefined. But anyway, can add it to unblock the PR

Reply:
The reason for the ask is that we are adding additional capabilities (of treating undefined as empty tag), and I prefer to have tests for such additions.

Also, all the tests that you had already added for `stringifyXML()` add coverage for the above, so its a win-win